Dockerfile

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:1.9.2
+FROM golang:1.12
```

Comment:
Maybe change this to `golang:1.12.6`?

@ravisantoshgudimetla and @aveshagarwal what do you think?

Reply:
Looks like Kubernetes 1.15 is built using `go1.12.5`.